magento/pwa-studio
# magento/magento2-page-builder#637: Form field/field group viewport scope

## Description

Changes were made to read non-media styles.

## Related Issue
magento/magento2-page-builder#637: Form field/field group viewport scope

## Acceptance
### Verification Steps
1. Create a page with page builder content type that has different min-height for desktop and mobile
2. Verify that the page works as before.


## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
